apps/src/p5lab/AnimationTab/ListItemThumbnail.jsx

```diff
@@ -83,7 +83,8 @@ export default class ListItemThumbnail extends React.Component {
       root: {
         border:
           'solid 2px ' +
-          (this.props.isSelected ? color.purple : color.light_gray)
+          (this.props.isSelected ? color.purple : color.light_gray),
+        maxHeight: this.state.previewSize + 8 + 'px'
```

Comment:
nit: i don't think 'px' is required here -- i think you can pass an integer value to `maxHeight` with inline styles.

will this squish any of the backgrounds? maybe i'm confused, but i thought this bug happened because some of the background assets aren't square, which means using the width to calculate the max-height could vertically squish any non-square assets.

Reply:
the background assets are square, previously they were just constrained on their width but not their height so it was letting them use their full height but squish their width to fit

Reply:
update: just kidding, we don't actually need to set max height, it isn't doing anything on the affected backgrounds.